grouper-ui/java/src/edu/internet2/middleware/grouper/grouperUi/beans/api/GuiGroup.java

```diff
@@ -648,4 +651,34 @@ public boolean isGrantAllAttrRead() {
   public boolean isGrantAllAttrUpdate() {
     return this.group.hasGroupAttrUpdate(SubjectFinder.findAllSubject());
   }
+  
+  /**
+   * if an attribute GrouperLoaderLdap is assigned to this group
+   * @return true
+   */
+  public boolean hasAttrDefNameGrouperLoaderLdap() {
+    boolean hasAttrLdap = false;
+    //on retrouve d'abord l'AttrDefName qui nous interesse
+    AttributeDefName grouperLoaderLdapName = GrouperDAOFactory.getFactory().getAttributeDefName().findByNameSecure(LoaderLdapUtils.grouperLoaderLdapName(), false); 
+    //On vérifie que l'attribute loader LDAP est bien affecté
+    if(grouperLoaderLdapName != null) {
+	  hasAttrLdap = this.group.getAttributeDelegate().hasAttribute(grouperLoaderLdapName);
+    }
+    return hasAttrLdap;
+  }
+
+  /**
+   * if an attribute GrouperLoader SQL is assigned to this group
+   * @return true

```

Comment:
Doesnt seem correct. You are also returning false, right? Can you document the method behavior?